src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -729,5 +733,25 @@ private static IEnumerable<string> GetClassesForType(Type type)
 				}
 			}
 		}
+
+		private InputCursor _protectedCursor = InputSystemCursor.Create(InputSystemCursorShape.Arrow);
+
+		protected InputCursor ProtectedCursor
+		{
+			get => _protectedCursor;
+			set
+			{
+				_protectedCursor = value;
+				Internal_SetProtectedCursor();
+			}
+
+		}
+
+		private void Internal_SetProtectedCursor()
```

Comment:
This should also consider if `_protectedCursor` is explicitly set to null.

Reply:
```suggestion
		private void SetProtectedCursorNative()
```

And yes we should keep the `_protectedCursor == null` unless explicitly set, and allow user to set it back to `null` (cf. My code suggestion below).